src/apps/backend/tests/modules/extract_purchase_order_history_request/test_extract_purchase_order_history_request_service.py

```diff
@@ -0,0 +1,41 @@
+from modules.account.account_service import AccountService
+from modules.account.types import CreateAccountParams
+from modules.extract_purchase_order_history_request.extract_purchase_order_history_request_service import (
+    PurchaseOrderHistorySerivce,
+)
+from modules.extract_purchase_order_history_request.types import (
+    ExtractPurchaseOrderHistoryParams,
+    ExtractPurchaseOrderHistoryRequestStatus,
+)
+from modules.vendor_account.types import CreateVendorAccountParams
+from modules.vendor_account.vendor_account_service import VendorAccountService
+from tests.modules.extract_purchase_order_history_request.base_test_extract_purchase_order_history_request import (
+    BaseTestExtractPurchaseOrderHistoryRequest,
+)
+
+
+class TestExtractPurchaseOrderHistoryRequestService(BaseTestExtractPurchaseOrderHistoryRequest):
```

Comment:
We should add a spec to ensure worker is queued with right params.

Reply:
Yes, I am doing that(WIP)